src/components/JobReorderModal.vue

```diff
@@ -0,0 +1,187 @@
+<template>
+  <ion-header>
+    <ion-toolbar>
+      <ion-buttons slot="start">
+        <ion-button @click="closeModal">
+          <ion-icon slot="icon-only" :icon="closeOutline" />
+        </ion-button>
+      </ion-buttons>
+      <ion-title>{{ $t('Export product threshold jobs') }}</ion-title>
+    </ion-toolbar>
+  </ion-header>
+
+  <ion-content>
+    <ion-reorder-group @ionItemReorder="doReorder($event)" disabled="false">
+      <ion-item v-for="job in jobs" :key="job.jobId">
+        <ion-label>{{ job.jobName }}</ion-label>
+        <ion-label>{{ job.runTime ? getTime(job.runTime) : "-"  }}</ion-label>
+        <ion-label>{{ timeTillJob(job.runTime)}}</ion-label>
+        <ion-icon :icon="checkmarkCircleOutline" color="success" v-if="successJobs.includes(job.jobId)" />
+        <ion-icon :icon="closeCircleOutline" color="danger" v-if="failedJobs.includes(job.jobId)" />
+        <ion-reorder />
+      </ion-item>
+    </ion-reorder-group>
+
+    <ion-fab vertical="bottom" horizontal="end" slot="fixed">
+      <ion-fab-button @click="save()" :disabled="this.failedJobs.length">
+        <ion-icon :icon="saveOutline" />
+      </ion-fab-button>
+    </ion-fab>
+  </ion-content>
+</template>
+
+<script lang="ts">
+import {
+  IonButtons,
+  IonButton,
+  IonContent,
+  IonFab,
+  IonFabButton,
+  IonHeader,
+  IonIcon,
+  IonItem,
+  IonLabel,
+  IonReorder,
+  IonReorderGroup,
+  IonTitle,
+  IonToolbar,
+  modalController
+} from '@ionic/vue';
+import { defineComponent } from 'vue';
+import { checkmarkCircleOutline, closeCircleOutline, closeOutline, saveOutline } from 'ionicons/icons';
+import { mapGetters, useStore } from 'vuex';
+import { DateTime } from 'luxon';
+import { JobService } from '@/services/JobService';
+import { hasError, showToast } from '@/utils';
+import logger from '@/logger';
+import { translate } from '@/i18n';
+
+export default defineComponent({
+  name: 'JobReorderModal',
+  components: {
+    IonButtons,
+    IonButton,
+    IonContent,
+    IonFab,
+    IonFabButton,
+    IonHeader,
+    IonIcon,
+    IonItem,
+    IonLabel,
+    IonReorder,
+    IonReorderGroup,
+    IonTitle,
+    IonToolbar,
+  },
+  data() {
+    return {
+      updatedJobsOrder: [] as any,
+      failedJobs: [] as any,
+      successJobs: [] as any,
+      jobs: (this as any).jobsForReorder,
```

Comment:
We could have modifiedJobs as clone copy of jobs & updated jobs and seqBeforeReorder might not be needed as the jobs property has this value